suites/tentacle/nvmeof/tier-3_nvmeof_scale_sub_ns.yaml

```diff
@@ -0,0 +1,364 @@
+# Test module to scale Single GW components - scale factor being 1 subsystem and 256 namespaces with 4 scale steps
```

Comment:
Please remove this suite as well as now we combined scale test suites with GW groups